### DISCUSSION TITLE
Crypto donations

### DISCUSSION THREAD

**Discussion by edxeth** [seerge/g-helper#3077] [Ideas]
Allow people to donate with major and known cryptocurrencies such as: Bitcoin, Ethereum, Solana, Dogecoin, USDT, and USDC. It takes nothing :)

P.S. the software is simply phenomenal, ASUS should learn from you guys!

**Comment by seerge** [comment]
@edxeth hello, thanks for a nice comment. Donations are extremely rare, so I don't plan to add more payment methods atm :)

**Reply by 0xKonstantin** [reply]
Hey @seerge I'm with Edxeth on this on, if you could allow crypto tips that would be amazing! To set up crypto payments you don't need to approve any personal information. A wallet is all you need, also Paypal has it built into their platform as well.

If you're up for I can always give you a quick run down would only take a few minutes and allow people all over the globe without bank access the ability to tip you <3
